### PR TITLE
Group page and member listing content

### PR DESCRIPTION
**Pull request by nikitr**
Show photos on member listing and group page includes group leaders

### CODE REVIEW

File: app/controllers/groups_controller.rb [hunk 1]
```diff
@@ -38,7 +41,12 @@ def member_listing
     @group = Group.friendly.find(params[:id])
   end
 
+  def settings
+    @group = Group.all
+  end
+
   def group_params
     params.require(:group).permit(:name, :description, :neighborhood_id)
```
Comment (hidesminimally) [inline app/controllers/groups_controller.rb]: @nikitr You need to add :kind to the params. Selecting the kind through the drop-down currently isn't working. Tested through creation

File: app/controllers/groups_controller.rb [hunk 2]
```diff
@@ -9,12 +9,15 @@ def index
   def show
     @post = Post.new
     @group = Group.friendly.find(params[:id])
+    @users = @group.users
```
Comment (hidesminimally) [inline app/controllers/groups_controller.rb]: @nikitr I don't think you're using `@user` variable. And, you don't need to set it here. `@group.users` will do in the html!

File: app/views/groups/show.html.erb [hunk 3]
```diff
@@ -4,13 +4,20 @@
 <h1>
   <%= @group.name %>
   <%= @group.description %>
+  <%= @group.kind %>
+  <% @leaders.each do |leader| %>
```
Comment (hidesminimally) [inline app/views/groups/show.html.erb]: @nikitr Do you need the `@leaders` variable here? Doesn't `@group.users.leaders` work?

File: app/views/groups/show.html.erb [hunk 4]
```diff
@@ -4,13 +4,20 @@
...
+    <li><%= leader.first_name %></li>
```
Comment (hidesminimally) [inline app/views/groups/show.html.erb]: @nikitr You want the full name. `leader.decorate.full_name`

File: app/views/groups/index.html.erb [hunk 5]
```diff
@@ -38,7 +38,7 @@
             <% end %>
           <% end %>
           <% if group.users.count > 4 %>
-            <%= '+' + group.users.count %>
+
```
Comment (hidesminimally) [inline app/views/groups/index.html.erb]: @nikitr Why was this deleted?

Comment (nikitr) [inline app/views/groups/index.html.erb]: @lilshim I think this was by accident, adding it back.

File: Gemfile [hunk 6]
```diff
@@ -5,6 +5,9 @@ gem "better_errors"
 gem 'aws-sdk'
 gem 'annotate'
 
+# Search functionality
+gem 'searchkick'
```
Comment (hidesminimally) [inline Gemfile]: @nikitr (nit) don't include things that aren't relevant for your feature. you should create a new branch for the elastic search + searchkick installment!
